xAPISGProcessor.ipynb

```diff
@@ -144,12 +116,18 @@
     "    %run vis/xAPISG-AlternativesSelectedQuestion.ipynb\n",
     "    resalt=vis_alternatives_selected_question()\n",
     "    alternativetab=VBox([resaltperplayer, resaltperalternative, resalt])\n",
-    "    tabs=widgets.Tab(children=[games_started_completed, videoseenskipped, playerprogress, completabletab,alternativetab])\n",
+    "    %run vis/xAPISG-ItemsInteracted.ipynb\n",
+    "    iteminteracted=VBox([vis_items_interacted_times_heatmap_interacted(),vis_items_interacted_times_interact()])\n",
+    "    %run vis/xAPISG-ItemsActionTypeInteracted.ipynb\n",
+    "    itemActionTypeinteracted=vis_items_action_type_interacted_times_interact()\n",
+    "    tabItem=VBox([iteminteracted,itemActionTypeinteracted])\n",
+    "    tabs=widgets.Tab(children=[games_started_completed, videoseenskipped, playerprogress, completabletab,alternativetab, tabItem])\n",
     "    tabs.set_title(0,\"Games started and Completed\")\n",
     "    tabs.set_title(1,\"Videos seen and skipped\")\n",
     "    tabs.set_title(2,\"Progress\")\n",
     "    tabs.set_title(3,\"Completable\")\n",
     "    tabs.set_title(4,\"Alternative\")\n",
+    "    tabs.set_title(5,\"Items interactions\")\n",
     "    tabs.selected_index=tabcheckboxes.selected_index\n",
```

Comment:
Missing tab for accessibles visualization - see previous comment about removing duplication of tabs.